GiniVision/Classes/GiniImageDocument.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  GiniImageDocument.swift
+//  GiniVision
+//
+//  Created by Enrique del Pozo Gómez on 9/4/17.
+//  Copyright © 2017 Gini GmbH. All rights reserved.
+//
+
+import Foundation
+import MobileCoreServices
+
+final public class GiniImageDocument: NSObject, GiniVisionDocument {
```

Comment:
Why does it have to be a `NSObject`?

Reply:
`NSItemProviderReading` needs to be implements by a class conforming `NSObjectProtocol`